[2023-06-10T01:22:39.648+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2019-03-02T06:30:00+00:00 [queued]>
[2023-06-10T01:22:39.663+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2019-03-02T06:30:00+00:00 [queued]>
[2023-06-10T01:22:39.664+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T01:22:39.664+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2023-06-10T01:22:39.665+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T01:22:39.681+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 06:30:00+00:00
[2023-06-10T01:22:39.690+0000] {standard_task_runner.py:54} INFO - Started process 1486 to run task
[2023-06-10T01:22:39.728+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_data', 'format_to_parquet_task', 'scheduled__2019-03-02T06:30:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmpxxtt4sug']
[2023-06-10T01:22:39.729+0000] {standard_task_runner.py:83} INFO - Job 60: Subtask format_to_parquet_task
[2023-06-10T01:22:39.730+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-10T01:22:39.917+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-10T01:22:39.963+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-10T01:22:40.064+0000] {task_command.py:384} INFO - Running <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2019-03-02T06:30:00+00:00 [running]> on host f41d61956f54
[2023-06-10T01:22:40.107+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-10T01:22:40.176+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=green_taxi_data
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T06:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T06:30:00+00:00
[2023-06-10T01:22:40.380+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_ingestion_gcs_multiFile_v02.py", line 33, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 886, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 895, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 114, in pyarrow.lib.check_status
OSError: zlib inflate failed: incorrect header check
[2023-06-10T01:22:40.392+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=green_taxi_data, task_id=format_to_parquet_task, execution_date=20190302T063000, start_date=20230610T012239, end_date=20230610T012240
[2023-06-10T01:22:40.403+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 60 for task format_to_parquet_task (zlib inflate failed: incorrect header check; 1486)
[2023-06-10T01:22:40.446+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-06-10T01:22:40.472+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
